src/databricks/labs/ucx/account.py

```diff
@@ -69,6 +70,84 @@ def sync_workspace_info(self):
             for installation in Installation.existing(ws, "ucx"):
                 installation.save(workspaces, filename=self.SYNC_FILE_NAME)
 
+    def create_account_level_groups(self):
+        acc_groups = self._get_account_groups()
+        all_valid_workspace_groups = self._get_valid_workspaces_groups()
+
+        for group_name, valid_group in all_valid_workspace_groups.items():
+            if group_name in acc_groups:
+                logger.info(f"Group {group_name} already exist in the account, ignoring")
+                continue
+
+            acc_group = self._ac.groups.create(display_name=group_name)
+
+            if len(acc_group.members) > 0:
+                self._add_members_to_acc_group(acc_group.id, group_name, valid_group)
+
+            logger.info(f"Group {group_name} created in the account")
+
+    def _add_members_to_acc_group(self, acc_group_id: str, group_name: str, valid_group: Group):
+        for chunk in self._chunks(valid_group.members, 20):
+            logger.debug(f"Adding 20 members to acc group {group_name}")
+            self._ac.groups.patch(
+                acc_group_id,
+                operations=[Patch(op=PatchOp.ADD, path="members", value=[x.as_dict() for x in chunk])],
+                schemas=[PatchSchema.URN_IETF_PARAMS_SCIM_API_MESSAGES_2_0_PATCH_OP],
+            )
+
+    def _chunks(self, lst, n):
+        """Yield successive n-sized chunks from lst."""
+        for i in range(0, len(lst), n):
+            yield lst[i : i + n]
+
+    def _get_valid_workspaces_groups(self) -> dict[str, Group]:
+        all_workspaces_groups: dict[str, Group] = {}
+
+        for workspace in self._workspaces():
+            client = self.client_for(workspace)
+            ws_group_ids = client.groups.list(attributes="id")
+            for group_id in ws_group_ids:
+                if not group_id.id:
+                    continue
+
+                full_workspace_group = client.groups.get(group_id.id)
+                group_name = full_workspace_group.display_name
+
+                if group_name in all_workspaces_groups:
+                    if self._has_same_members(all_workspaces_groups[group_name], full_workspace_group):
+                        logger.info(f"Workspace group {group_name} already found, ignoring")
+                        continue
+
+                    logger.warning(
```

Comment:
```suggestion
                    if prompts.confirm(
```
It has to be a conscious choice of account admin to do so

Reply:
Much better indeed